### PR TITLE
Upgrade dependencies required to get us to Apollo Client 3.1.3

### PR DESCRIPTION
**Pull request by nicolas6422**
Upgrade dependencies required to get us to Apollo Client 3.1.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "test": "./scripts/test.sh",
     "test-compile": "./scripts/test-compile.sh",
     "test-style": "./scripts/test-style.sh",
-    "test-unit": "./scripts/test-unit.sh",
+    "test-unit": "DEBUG=jest ./scripts/test-unit.sh",
```
Comment (nicolas6422) [inline package.json:32]: remove

File: src/Cache.ts [hunk 2]
```diff
@@ -128,7 +128,11 @@ export class Cache implements Queryable {
       callback(transaction);
     } catch (error) {
       if (tracer.transactionEnd) {
-        tracer.transactionEnd(error.toString(), tracerContext);
+        if (error instanceof Error) {
+          tracer.transactionEnd(error.toString(), tracerContext);
+        } else {
+          tracer.transactionEnd('unknown error', tracerContext);
```
Comment (arama-convoy) [inline src/Cache.ts:134]: you encountered a situation where a non-error exception was being thrown?

File: src/util/store.ts [hunk 3]
```diff
@@ -19,24 +19,29 @@ export function valueFromNode(
   onVariable: VariableValue = defaultValueFromVariable,
 ): any {
   switch (node.kind) {
-    case 'Variable':
+    case Kind.VARIABLE:
```
Comment (arama-convoy) [inline src/util/store.ts:22]: nice!

File: package.json (hunk 1 above)
Comment (arama-convoy) [reply package.json]: https://media.giphy.com/media/xT9DPr4VjeCgeiLoMo/giphy.gif

**Review by arama-convoy — APPROVED** [review]

File: src/Cache.ts (hunk 2 above)
Comment (nicolas6422) [reply src/Cache.ts:134]: i updated typescript or tslib and had some errors because of that, this was one of them

**Pull request merged**